src/libs/blueprint/CMakeLists.txt

```diff
@@ -112,14 +112,27 @@ if(FORTRAN_FOUND)
 
 endif()
 
+set(conduit_blueprint_deps conduit)
+
+###################
+# Add zfp interface
+# if zfp was found
+###################
+
+if(ZFP_FOUND)
+  list(APPEND blueprint_headers "conduit_blueprint_zfparray.hpp")
+  list(APPEND blueprint_sources "conduit_blueprint_zfparray.cpp")
+  list(APPEND conduit_blueprint_deps zfp)
```

Comment:
We want to keep blueprint itself dependency free.  
I see zfp is included in the cpp-source, but do we actually need it here? 
I don't think its being used